tools/openms/DatabaseFilter.xml

```diff
@@ -75,6 +75,8 @@ python3 '$__tool_directory__/fill_ctd.py' '@EXECUTABLE@.ctd' '$args_json' '$hard
   <help><![CDATA[Filters a protein database (FASTA format) based on identified proteins
 
 
-For more information, visit http://www.openms.de/doxygen/release/2.6.0/html/UTILS_DatabaseFilter.html]]></help>
+For more information, visit http://www.openms.de/doxygen/release/2.6.0/html/UTILS_DatabaseFilter.html
+
+tool catecory: Utilities]]></help>
```

Comment:
I don't think we should do this. This should be really on the Admin level or part of the EDAM ontology, no?

Reply:
> I don't think we should do this.

Maybe :) Its easy to remove. 

> This should be really on the Admin level or part of the EDAM ontology, no?

Do you have something specific in mind. The idea was to (kind of) simulate what you have in knime where you have these categories of tools. Don't think that the tools can easily be mapped to EDAM, or?

Reply:
If would not mirror KNIME just because ... ideally we group tools in meaningful categories and if EDAM offers those we have a nice ontology that we can use. I would imagine that most of them can be mapped to EDAM ... but this will still take time.

Reply:
Agreed. I undid the changes.